lib/led/led.js

```diff
@@ -156,8 +163,16 @@ function Led(opts) {
 
   Object.defineProperties(this, controller);
 
+  /**
+    * @deprecated Will be deleted in version 0.11.8. Use 'inverted' instead.
+  */
+  if (typeof opts.isAnode !== 'undefined' && opts.isAnode !== null) {
+    console.log("isAnode is deprecated. Use 'inverted' instead");
+    opts.inverted = opts.isAnode;
+  }
+
   var state = {
-    isAnode: opts.isAnode,
+    inverted: opts.inverted,
```

Comment:
We should not use "inverted" because it is ambiguous. An LED is either common anode or common cathode. There is no such state as "inverted" because there is no default state to invert.

Reply:
Linking this [issue](https://github.com/rwaldron/johnny-five/issues/1026) wherein @soundanalogous gives us the labels ```sourcedrive``` and ```syncdrive``` which makes sense to me.

Reply:
So will it continue to be isAnode?

Reply:
As I understand it "common anode" and "common cathode" are RGB LED properties. This change is specifically for single color LED's. So I would think @soundanalogous's suggestion of "syncdrive" (from #1026) would be the appropriate property here.

I also propose that instead of adding this condition to every controller we add a "syncdrive" device type that has a new ```getValue()``` or maybe just ```value()``` method.

Lastly, there is a [new property](https://github.com/firmata/firmata.js/pull/149) on the io object (in firmata at least) that allows us to know the resolution of pwm writes (i.e. 10-bit instead of 8-bit). We should leverage that here rather than assuming 8-bit.

Reply:
This is still an issue, but it's sat for a while. Perhaps we can settle on a label for the property at JSCONF this year?